_includes/content/private-registry-image-path.md

```diff
@@ -0,0 +1,128 @@
+
+1. Use the following commands to pull the required {{site.prodname}} images.
+
+   ```bash
+   docker pull {{ operator.registry }}/{{ operator.image }}:{{ operator.version }}
+   {% for component in site.data.versions.first.components -%}
+   {% if component[1].image -%}
+   {% if component[1].registry %}{% assign registry = component[1].registry | append: "/" %}{% else %}{% assign registry = page.registry -%} {% endif -%}
+   docker pull {{ registry }}{{ component[1].image }}:{{component[1].version}}
+   {% endif -%}
+   {% endfor -%}
+   ```
+
+1. Retag the images with the name of your private registry `$PRIVATE_REGISTRY` and `$IMAGE_PATH`.
+
+   ```bash
+   docker tag {{ operator.registry }}/{{ operator.image }}:{{ operator.version }} $PRIVATE_REGISTRY/$IMAGE_PATH/{{ operator.image | split: "/" | last }}:{{ operator.version }}
+   {% for component in site.data.versions.first.components -%}
+   {% if component[1].image -%}
+   {% if component[1].registry %}{% assign registry = component[1].registry | append: "/" %}{% else %}{% assign registry = page.registry -%} {% endif -%}
+   docker tag {{ registry }}{{ component[1].image }}:{{component[1].version}} $PRIVATE_REGISTRY/$IMAGE_PATH/{{ component[1].image | split: "/" | last }}:{{component[1].version}}
+   {% endif -%}
+   {% endfor -%}
+   ```
+
+1. Push the images to your private registry.
+
+   ```bash
+   docker push $PRIVATE_REGISTRY/$IMAGE_PATH/{{ operator.image | split: "/" | last }}:{{ operator.version }}
+   {% for component in site.data.versions.first.components -%}
+   {% if component[1].image -%}
+   docker push $PRIVATE_REGISTRY/$IMAGE_PATH/{{ component[1].image | split: "/" | last}}:{{component[1].version}}
+   {% endif -%}
+   {% endfor -%}
+   ```
+
+   > **Important**: Do not push the private {{site.prodname}} images to a public registry.
+   {: .alert .alert-danger}
+
+#### Run the operator using images from your private registry image path
+
+Before applying `tigera-operator.yaml`, modify registry references to use your custom registry:
+
+```bash
+{% if page.registry != "quay.io/" -%}
+sed -ie "s?{{ page.registry }}.*/?$PRIVATE_REGISTRY/$IMAGE_PATH/?" tigera-operator.yaml
+{% endif -%}
+sed -ie "s?quay.io.*/?$PRIVATE_REGISTRY/$IMAGE_PATH/?" tigera-operator.yaml
+```
+{% comment %} The second 'sed' should be removed once operator launches Prometheus & Alertmanager {% endcomment %}
+
+Next, ensure that an image pull secret has been configured for your custom registry. Set the enviroment variable `PRIVATE_REGISTRY_PULL_SECRET` to the secret name.
+Then add the image pull secret to the operator deployment spec:
+
+```bash
+sed -ie "/serviceAccountName: tigera-operator/a \      imagePullSecrets:\n\      - name: $PRIVATE_REGISTRY_PULL_SECRET"  tigera-operator.yaml
+```
+
+If you are installing Prometheus operator as part of {{ site.prodname }}, then before applying `tigera-prometheus-operator.yaml`, modify registry references to use your custom registry:
```

Comment:
This doesn't apply to OSS Calico (prometheus-operator)

Reply:
I don't why the doc was architected this way. Generally, I don't touch the install docs. I will ask Erik about the include versus inline comment.

Reply:
This comment is only saying that this line about prometheus isn't valid for OSS Calico. Line 62 should be removed.